src/features/device-logs/lib/backends/redis.ts

```diff
@@ -270,7 +270,7 @@ export class RedisBackend implements DeviceLogsBackend {
 		payload: string,
 	): Promise<RedisDeviceLog | undefined> {
 		try {
-			let decompressedBuffer = Buffer.from(payload, BUFFER_ENCODING);
+			let decompressedBuffer: Buffer = Buffer.from(payload, BUFFER_ENCODING);
```

Comment:
`Buffer.from` returns `Buffer<ArrayBuffer>, but `compression.uncompress` returns `Promise<Buffer>`, and now `Buffer` has a default generic of `Buffer<ArrayBufferLike>`, so we had to define decompressedBuffer with the wider `Buffer<ArrayBufferLike>`, which then eslint simplifies to `Buffer`, since `ArrayBufferLike` is the default.